sdk/typescript/introspector/test/invoke.spec.ts

```diff
@@ -51,31 +50,59 @@ describe("Invoke typescript function", function () {
     // Load function
     await load(files)
 
+    const scanResult = scan(files)
+
     // Mocking the fetch from the dagger API
     const input = {
-      objectName: "Bar",
-      methodName: "exec",
-      state: {},
-      inputs: {
+      parentName: "Bar",
+      fnName: "exec",
+      parentArgs: {},
+      fnArgs: {
         // string[]
         cmd: ["echo", "-n", "hello world"],
```

Comment:
The indentation looks a bit off here. Isn't the linter covering this?

Reply:
Yes it does :D